aday2dream/src/main/kotlin/com/aday2dream/aday2dream/service/AudioFileService.kt

```diff
@@ -0,0 +1,53 @@
+package com.aday2dream.aday2dream.service
+
+import com.aday2dream.aday2dream.model.AudioFile
+import com.aday2dream.aday2dream.repository.AudioFileRepository
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.springframework.web.multipart.MultipartFile
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+import kotlin.io.path.exists
+
+@Service
+class AudioFileService(
+    private val audioFileRepository: AudioFileRepository,
+    @Value("\${audiofile.storage.directory}")private val uploadDir: String
```

Comment:
so you will save the files locally on the disc ... when this gets deployed and you need to save those audio files, it might turn out to be costly
you need to be aware of this major Con with the implementation decision you have made